pybamm/simulation.py

```diff
@@ -0,0 +1,236 @@
+import pybamm
+import numpy as np
+import copy
+
+
+class Simulation:
+    """A Simulation class for easy building and running of PyBaMM simulations.
+
+    Parameters
+    ----------
+    model : :class:`pybamm.BaseModel`
+        The model to be simulated
+    """
+
+    def __init__(self, model):
```

Comment:
would be good to (optionally) pass params, geometry etc here when you create the simulation

Reply:
Agreed

Reply:
Would it be reasonable to allow to optionally pass another parameter set?

Reply:
@ferranbrosa do you mean something different from @rtimms suggestion?

Reply:
Sorry, no. I read it too fast so I only got geometry (missed params).